kitchen.yml

```diff
@@ -3,46 +3,43 @@ driver:
 
 provisioner:
   name: chef_zero
+  enforce_idempotency: true
+  multiple_converge: 2
   deprecations_as_errors: true
   chef_license: accept-no-persist
 
 verifier:
   name: inspec
 
 platforms:
-  - name: amazonlinux
-    driver_config:
-      box: mvbcoding/awslinux
   - name: amazonlinux-2
-  - name: centos-6
   - name: centos-7
   - name: centos-8
-  - name: debian-8
-    run_list: apt::default
   - name: debian-9
     run_list: apt::default
```

Comment:
This should be removed from any debian-based host in the regular kitchen.yml file.